client/js/controllers/loginController.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+featureToggleFrontend.controller('LoginController', ['$scope', function ($scope, $location) {
+    $scope.authProvider = getUrlParameter('authProvider');
+    
+    function getUrlParameter(name){
```

Comment:
yes, i'll update the code to get the config from: \client\configuration\config.js